start.php

```diff
@@ -54,8 +53,8 @@ function mentions_get_views() {
  * @return string
  */
 function mentions_rewrite($hook, $entity_type, $returnvalue, $params) {
-
 	$regexp = mentions_get_regex();
+
```

Comment:
i did not make these changes thinking they would be pulled in to the main code, so i coded them according to my own preference.